cedar-policy-validator/src/cedar_schema/err.rs

```diff
@@ -201,6 +204,15 @@ impl Display for ParseErrors {
     }
 }
 
+impl IntoIterator for ParseErrors {
+    type Item = ParseError;
+    type IntoIter = Chain<Once<ParseError>, vec::IntoIter<ParseError>>;
```

Comment:
What's an idiomatic type we should use here? The current one seems overly complicated.

Reply:
Eh it's a combination of types that are all in the stdlib. Unless we want to re-implement the underlying iterators this seems like the best.

Reply:
Another option is `type IntoIter = <NonEmpty as IntoIterator>::IntoIter;`

Reply:
If this is part of our public API then I think we should expose a nicer iterator type. In other parts of our API we use a wrapper struct that holds the actual iter in a private field.